### ISSUE TITLE
Build

### ISSUE THREAD

**Issue by Andres8427** [abhioncbr/docker-superset#20]
Hello, first of all I congratulate you and I thank you for the contribution of your repository, it has made it much easier for me to start with the concept tests that I must do on Superset.

I'm not the most expert in Docker so the question may sound a little stupid.

I managed to start the container using docker-compose and it works perfectly, however I must make some adjustments from the beginning so I modified the Dockerfile and generated the image with the "docker build" as it is specified in the documentation. However, I can not get the container to start, it always remains in an error state. I'm not clear about the command that I should use. It must be this:

"cd docker-superset & & docker run -p 8088: 8088 -v config: / home / superset / config / abhioncbr / docker-superset: <version-tag> cluster server <superset_metadata_db_url> <redis_url>"

the parameter "superset_metadata_db_url" is the host of the metadata DB with the structure user @ host .....?

When running the "docker build" does not generate the redis container, I must generate it separately and in "redis_url" I send the URL of this one?

Basically what I need is to use the Superset container but to automatically execute a Sh that modifies the Superset configuration, also creates some folders in the container and copies some monitoring routines, but that allows us to use Superset updates, that I thought about modifying the Dockerfile and executing the Sh in the "docker-entrypoint.sh", in that case the best option is to use the docker build of your container? or is there some other simpler way?

I would very much appreciate any help you can give me!

Thank you.
